index.bs

```diff
@@ -1957,6 +1977,11 @@ To <dfn>obtain a randomized source response pick rate</dfn> given a [=randomized
 1. Let |numPossibleValues| be the [=set/size=] of |possibleValues|.
 1. Return |numPossibleValues| / (|numPossibleValues| - 1 + e<sup>|epsilon|</sup>).
 
+To <dfn>obtain pick rate for a source with attribution scope</dfn> given a positve 32-bit integer |numSourceStates|, a positve 32-bit integer |numEventStates|, a positve 32-bit integer |numUsedScopes|, a positive and a double |epsilon|:
```

Comment:
`numSourceStates` seems a bit confusing. Until now, the spec has used num trigger states for this, no?

Reply:
Let's go with how the ARA spec has been referring to this concept.

Reply:
Done.